features/taxation/applying_taxes/applying_correct_taxes_for_order_with_promotion_giving_discount_for_whole_order.feature

```diff
@@ -0,0 +1,42 @@
+@applying_taxes
+Feature: Apply correct taxes on order item level when order has applied a promotion with discount for whole order
+    In order to pay proper amount when buying goods
+    As a Visitor
+    I want to have correct taxes applied to my order when it has applied a promotion
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And default tax zone is "US"
+        And the store has "US VAT" tax rate of 23% for "Clothes" within the "US" zone
+        And the store has "Low VAT" tax rate of 10% for "Mugs" within the "US" zone
+        And the store has a product "PHP T-Shirt" priced at "$10.00"
+        And it belongs to "Clothes" tax category
```

Comment:
Can we just use 2 products instead of 4? 1 for `Properly rounded up tax for single product`, 1 for `Properly rounded down tax for single product`, each from different tax category, and both of them in `Properly rounded taxes for multiple products`